docs/configuration.asciidoc

```diff
@@ -933,3 +933,33 @@ During startup the Node.js agent queries the local environment to determine whet
 - `none`: Do not query for any cloud provider information.
 
 If the value is not one of the five listed above, the agent will use the value of `auto`.
+
+[[ignore_message_queues]]
+==== `ignoreMessageQueues`
+* *Type:* Array
+* *Default:* `[]`
+* *Env:* `ELASTIC_IGNORE_MESSAGE_QUEUES`
+
```

Comment:
Are you adding support for central config later?

Reply:
Ah -- good catch.  I'd missed/assumed that this wasn't a configuration intended for central config, but re-reading the spec confirms it is.   

I've now added it to the `CENTRAL_CONFIG` array in `lib/config.js` and updated these docs and tested the configuration with a mock APM server.   I'll also need to cargo cult this PR in Kibana to make this configuration a central config. https://github.com/elastic/kibana/pull/85655